RUFAS/output_manager.py

```diff
@@ -187,9 +189,21 @@ def add_error(self, name: str, msg: str, info_map: Dict[str, Any]) -> None:
         info_map["suffix"] : str, optional
             If present, gets appended to the key
         """
+        self._add_timestamp_to_info_map(info_map)
         key = self._generate_key(name, info_map)
         self._add_to_pool(self.errors_pool, key, msg, info_map)
 
+    def _get_timestamp(self, include_millis: bool = False) -> str:
+        """Produce the current system time as a timestamp string."""
+        base_timestamp_str: str = "%d-%b-%Y_%a_%H-%M-%S"
+        timestamp_format_string: str = f"{base_timestamp_str}.%f" if include_millis else base_timestamp_str
+        return datetime.datetime.now().strftime(timestamp_format_string)
+
+    def _add_timestamp_to_info_map(self, info_map: Dict[str, Union[str, bool]]) -> None:
```

Comment:
```suggestion
    def _add_timestamp_to_info_map(self, info_map: Dict[str, Any]) -> None:
```
Why Union[str, bool]?

Reply:
I copied the type from another method in the OutputManager. I removed the method completely so no longer applies.